Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8
+FROM node:10
```

Comment:
I think we should be able to use `node:10-alpine` here

Reply:
Done!